core/templates/components/ck-editor-helpers/ck-editor-copy-content.service.ts

```diff
@@ -122,26 +127,36 @@ export class CkEditorCopyContentService {
     let elementTagName = (
       containedWidgetTagName || element.tagName.toLowerCase());
     let html = element.outerHTML;
-
+    html = html.replace(/<!--[^>]*-->/g, '').trim();
     if (!containedWidgetTagName) {
       editor.insertHtml(html);
     } else {
       const widgetName = elementTagName.replace('-noninteractive-', '');
 
       // Look for x-with-value="y" to extract x and y.
-      //  Group 1 (\w+): Any word containing [a-zA-Z0-9_] characters. This is
-      //    the name of the property.
+      //  Group 1 ([\w-]+): Any word containing [a-zA-Z0-9_-] characters. This
+      //    is the name of the property. (Note that this will also catch
+      //    ng-reflect-* which is added by angular).
```

Comment:
```suggestion
      // Group 1 ([\w-]+): Any word containing [a-zA-Z0-9_-] characters. This
      // is the name of the property. (Note that this will also catch
      // ng-reflect-* which is added by angular).
```

Reply:
Why are we expecting to catch `ng-reflect-*` here? Isn't it only expected to catch interaction-specific attributes?

Reply:
Yes it is. But then I am also discarding the ng-reflect-* below. I can't precisely recall what went wrong with the old regex but I had to change it. (Line 151-159)

Reply:
On line 144 we are firstly catching these new attributes and then ignoring it on line 158, can we just ignore it through the Regis check!

Also, in case it’s a for some “special” case then I’ll would highly suggest writing a code comment here!